proposals/2021-02-enable-ibc-transfers/README.md

```diff
@@ -0,0 +1,12 @@
+# Enable IBC Transfers
+
+The Proposal enables transfering and recieving assets using the IC20 protocol on the Cosmos Hub. If this proposal passes, there will be IBC assets available in the Bank module of the Hub and ATOM will be available on Zones connected over IBC.
+Iqlusion believes that the IBC software is suffciently stable for small amounts of value transfer. We expect there to be issues with stuck funds and UX confusion but overcoming these issues will only happen once IBC is live.
+
+## Security Model
+
+Tendermint full nodes produce agreement under the assumption that at most ⅓ of the voting power held by validators is Byzantine.
+
+## IBC
+IBC is a protocol for authenticated message passing between heterogeneous sovereign blockchains. IBC requires trusting that chains on both sides of the connections operate within their security model.\n Incentive Security Extensions
```

Comment:
```suggestion
IBC is a protocol for authenticated message passing between heterogeneous sovereign blockchains. IBC requires trusting that chains on both sides of the connections operate within their security model.

## Incentive Security Extensions

```

Reply:
```suggestion
IBC is a protocol for authenticated message passing between homogeneous (Cosmos SDK/Tendermint) sovereign blockchains. IBC requires trusting that chains on both sides of the connections operate within their security model.\n Incentive Security Extensions
```

Reply:
Actually @johnniecosmos its not all just Cosmos/Tendermint chains. Other consensus mechanisms will be supported as well as traditional databases.

Reply:
Hey man! Just pointing to the distinction between homogeneous (Cosmos SDK/Tendermint) and heterogeneous (ETH, BTC etc).